coalib/bearlib/spacing/SpacingHelper.py

```diff
@@ -86,11 +86,28 @@ def replace_spaces_with_tabs(self, line: str):
         :param line: The string with spaces to replace.
         :return:     The converted string.
         """
+        def previous_whitespace():
+            # Find the previous real character, and remaining chars to fill tab
+            non_whitespace_position = tabless_position - currspaces
+            tab_fill = non_whitespace_position % self.tab_width
+
+            if tab_fill and tab_fill + currspaces >= self.tab_width:
+                whitespace = "\t"
+                remaining_spaces = currspaces - (self.tab_width - tab_fill)
+            else:
+                whitespace = ""
+                remaining_spaces = currspaces
```

Comment:
:+1: our bear would work for all coalang supported languages though because it's really simple to do. But we could use flake-quotes in the meantime. If you find it disturbing, file an issues. Otherwise we'll just wait for it to happen somewhen.